packages/next/src/server/config.ts

```diff
@@ -721,6 +721,10 @@ function assignDefaults(
         '*': 'modularize-import-loader?name={{ member }}&from=default&as=default&join=../esm/icons/{{ kebabCase member }}!lucide-react',
       },
     },
+    '@fortawesome/?((pro-|free-|sharp-)[a-z-]+)': {
```

Comment:
We should look into version locking these as these are locked to the Next.js version and if a library ever changes their structure old versions will just stop working with newer versions of the package. 

At the least we should move user provided modularize config at the bottom so it can override our defaults. 

x-ref: [slack thread](https://vercel.slack.com/archives/C04DUD7EB1B/p1691351150296649)